test/integration/commands/excel2json/test_resources.py

```diff
@@ -183,9 +189,8 @@ class TestValidateWithSchema:
     lambda _: e2j._validate_resources([])
 
     def test_invalid_super(self) -> None:
-        expected_msg = (
-            "\nThe Excel file 'testdata/invalid-testdata/excel2json/resources-invalid-super.xlsx' "
-            "did not pass validation.\n"
+        expected_msg = regex.escape(
+            "\nThe Excel file 'resources.xlsx' did not pass validation.\n"
```

Comment:
I wonder if it is a good idea to always say 'resources.xlsx', even if the actual name could be different. It would be better to use the real file name. (Even if currently, it will always be 'resources.xlsx', in the future it might be different).